specification/resources/resource-manager/Microsoft.Solutions/stable/2019-07-01/managedapplications.json

```diff
@@ -0,0 +1,1949 @@
+      }
+    },
+    "ApplicationArtifact": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The managed application artifact name."
+        },
+        "uri": {
+          "type": "string",
+          "description": "The managed application artifact blob uri."
+        },
+        "type": {
+          "$ref": "#/definitions/ApplicationArtifactType",
+          "description": "The managed application artifact type."
+        }
+      },
+      "description": "Managed application artifact."
+    },
+    "ApplicationProviderAuthorization": {
+      "properties": {
+        "principalId": {
+          "type": "string",
+          "description": "The provider's principal identifier. This is the identity that the provider will use to call ARM to manage the managed application resources."
+        },
+        "roleDefinitionId": {
+          "type": "string",
+          "description": "The provider's role definition identifier. This role will define all the permissions that the provider must have on the managed application's container resource group. This role definition cannot have permission to delete the resource group."
+        }
+      },
+      "required": [
+        "principalId",
+        "roleDefinitionId"
+      ],
+      "description": "The managed application provider authorization."
+    },
+    "ErrorResponse": {
+      "description": "Error response indicates managed application is not able to process the incoming request. The reason is provided in the error message.",
+      "type": "object",
+      "properties": {
+        "httpStatus": {
+          "description": "Http status code.",
+          "type": "string"
+        },
+        "errorCode": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "errorMessage": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "JitRequestDefinition": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/JitRequestProperties",
+          "description": "The JIT request properties."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "description": "Information about JIT request definition."
+    },
+    "JitRequestDefinitionListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/JitRequestDefinition"
+          },
+          "description": "The array of Jit request definition."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to use for getting the next set of results."
+        }
+      },
+      "description": "List of managed applications."
+    },
+    "JitRequestProperties": {
+      "properties": {
+        "ApplicationResourceId": {
```

Comment:
Is the intent to have a LinkedAccessCheck on this field?

Reply:
Updated Microsoft.Solutions manifest. Thanks.